metabase/metabase
# command palette trigger in app bar search

Closes https://github.com/metabase/metabase/issues/40735

### Description
Introduces another way to open the command palette via the search bar, and removes the existing CTA at the bottom of the recents panel. Does not appear when embedded

### How to verify
Click the search input in the app bar. You should see a small button showing how to open the command palette with a tooltip on hover

*Update*
One thing I should call out here is that I've updated the display props for the search bar to use "more flex". I did test locally that things still worked as expected in Mobile, but might be worth double checking

### Demo
![chrome_yAHogkjUoH](https://github.com/metabase/metabase/assets/1328979/43bc2c4f-1601-4cb4-b06d-e1d8fdb623b5)

### Checklist
